lib/private/User/Database.php

```diff
@@ -448,7 +466,7 @@ public function userExists($uid) {
 	 */
 	public function getHome(string $uid) {
 		if ($this->userExists($uid)) {
-			return \OC::$server->getConfig()->getSystemValueString('datadirectory', \OC::$SERVERROOT . '/data') . '/' . $uid;
+			return \OCP\Server::get(IConfig::class)->getSystemValueString('datadirectory', \OC::$SERVERROOT . '/data') . '/' . $uid;
```

Comment:
```suggestion
			return $this>config->getSystemValueString('datadirectory', \OC::$SERVERROOT . '/data') . '/' . $uid;
```
You added config to properties

Reply:
```suggestion
			return $this>config->getSystemValueString('datadirectory', \OC::$SERVERROOT . '/data') . '/' . $uid;
```